boards/arm/ev11l78a/ev11l78a_defconfig

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_SAMD20=y
+CONFIG_SOC_PART_NUMBER_SAMD20E16=y
+CONFIG_BOARD_EV11L78A=y
+CONFIG_BUILD_OUTPUT_HEX=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_GPIO=y
+CONFIG_SOC_ATMEL_SAMD_OSC8M=y
+CONFIG_SOC_ATMEL_SAMD_OSC8M_AS_MAIN=y
+CONFIG_ADC=y
```

Comment:
This should probably not be here, but rather be enabled by specific samples or applications requiring the ADC.